src/Plugin/Block/PatternkitBlock.php

```diff
@@ -551,7 +551,8 @@ public function build() {
         foreach ($tokens as $token) {
           $placeholder = preg_replace("/[^a-z]/", '', $token);
           $template_context[$placeholder] = $tokenized[$token];
-          $template = str_replace($token, $placeholder, $template);
+          // Wrap the template in Twig variable brackets for inline rendering.
+          $template = '{{ ' . str_replace($token, $placeholder, $template) . ' }}';
```

Comment:
A few things, this breaks when:

- multiple tokens are in the user string
- Twig write brackets are already in the user string
- Twig filters are being used without concatenation operators

Reply:
See https://github.com/drupal-pattern-lab/patternkit/pull/56/files#diff-fa66c899860171cee9069390ef20ee4d42e3fcec741eb2650b25fc63ac89dd77R566